assets/js/state/sapSystems.js

```diff
@@ -183,6 +192,15 @@ export const sapSystemsListSlice = createSlice({
         { deregistering: false }
       );
     },
+    updateDatabaseInstanceAbsentToSAPSystem: (state, { payload: instance }) => {
```

Comment:
What about putting this right after the other? as they are related and to avoid scrolling

Reply:
done